### PR TITLE
added prerequisites requirement for pages subject to change

### PR DESCRIPTION
**Pull request by gorondan**
## Wiki PR Checklist

Thank you for contributing to the Protocol Wiki! Before you open a PR, make sure to read [information for contributors](https://epf.wiki/#/contributing) and take a look at following checklist:

- [x] Describe your changes, substitute this text with the information
added prerequisites requirement for pages subject to change, in contributing.md
- [x] If you are touching an existing piece of content, ask the original creator for review

### CODE REVIEW

**Comment by gorondan** [comment]
fixes #249

File: docs/contributing.md [hunk 1]
```diff
@@ -87,6 +87,21 @@ Usage of images and visualizations is encouraged. If you are using an image crea
 
 Feel free to use [emojis](https://docsify.js.org/#/emoji?id=emoji) or [icons](https://icongr.am/fontawesome) where it fits, for example in blockquotes. 
 
+### Prerequisites
+Wiki pages that are subject to future updates, i.e. pages covering active research topics require prerequisites to be added on top of the pages:
```
Comment (raxhvl) [inline docs/contributing.md:91]:
```suggestion
#### Warning stub for active research topics
Wiki pages that are subject to future updates, i.e. pages covering active research topics require prerequisites to be added on top of the pages:
```

File: docs/contributing.md [hunk 2]
```diff
@@ -87,6 +87,21 @@ Usage of images and visualizations is encouraged. If you are using an image crea
...
+
+* Warning message artifact using the following format:
+> [!WARNING]
+> This document covers an active area of research, may be outdated at time of reading and subject to future updates, as the design space around (*active research topic, e.g. Inclusion Lists*) evolves.
+
+* Roadmap tracker artifact using the following format:
```
Comment (raxhvl) [inline docs/contributing.md:97]:
```suggestion
#### Rodmap format
Roadmap tracker artifact using the following format:
```

**Review by raxhvl — CHANGES_REQUESTED** [review]
Thanks for the quick response. I have added headings to both sections so that it can be referenced easily.

File: docs/contributing.md (hunk 1 above)
Comment (gorondan) [reply docs/contributing.md:91]: thank you for the addition <3

File: docs/contributing.md (hunk 2 above)
Comment (gorondan) [reply docs/contributing.md:97]: One thing about roadmap tracker: if we go for the proposed changes, it's not clear imo if the tracker should be added to all pages or just to research pages([acc. to this discussion](https://github.com/eth-protocol-fellows/protocol-studies/issues/179#issuecomment-2056404495)). That's why in the initial commit, both artifacts were presented under `research` pages

**Pull request merged**